Package.resolved

```diff
@@ -19,6 +28,24 @@
           "version": "0.0.1"
         }
       },
+      {
+        "package": "swift-case-paths",
+        "repositoryURL": "https://github.com/pointfreeco/swift-case-paths",
+        "state": {
+          "branch": null,
+          "revision": "ed1838ab4fa5d47db8aa640736dd5b7672548873",
+          "version": "0.1.2"
+        }
+      },
+      {
+        "package": "swift-composable-architecture",
+        "repositoryURL": "https://github.com/pointfreeco/swift-composable-architecture",
+        "state": {
+          "branch": null,
+          "revision": "6d4c40d8c2faea519e090d1c342cbaf7bd7e9b9a",
+          "version": "0.10.0"
```

Comment:
FYI at the moment TCA has some breaking changes.

Reply:
I see they've made [a number of releases](https://github.com/pointfreeco/swift-composable-architecture/releases) recently to add support for forms and `TextState`, but am not noticing any breaking changes to what we're using here.